Majasdarbs/10-Galadarbs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "checkifsimilar",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build-css": "tailwind build -i src/styles.css -o public/styles.css --watch",
```

Comment:
-o dist/styles.css
Remove styles.css in public.
Css will be auto generated based on src/styles.css + classes that are used in js/html files